apps/docs/src/app/core/component-docs/dialog/dialog-docs.component.html

```diff
@@ -0,0 +1,194 @@
+
+<description>
+    Default values used by the Dialog can be specified by providing <code>DIALOG_DEFAULT_CONFIG</code> token value.<br/><br/>
+    <code>
+        @NgModule({{'{'}}
+            provide: [
+                {{'{'}}provide: DIALOG_DEFAULT_CONFIG, useValue: customConfig{{'}'}}
+            ]
+        {{'}'}}
+    </code>
+</description>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogPosition'" [componentName]="'dialog'">
+    Dialog positioning
+</fd-docs-section-title>
+<description>
+    The placement of the Dialog on the screen can be customized through the <code>DialogConfig.position</code> property.
+</description>
+<component-example>
+    <fd-dialog-position-example></fd-dialog-position-example>
+</component-example>
+<code-example [exampleFiles]="dialogPosition"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogMobile'" [componentName]="'dialog'">
+    Dialog on mobile devices
+</fd-docs-section-title>
+<description>
+    To display Dialog component in mobile mode set use <code>DialogConfig.mobile</code> property.
+    Dialog on mobile devices takes full viewport and adjusts its header and footer. To create additional spacing around
+    Dialog use <code>DialogConfig.mobileOuterSpacing</code> property.
+    <br/><br/>
+    To achieve full-screen dialog without mobile mode features use <code>DialogConfig.fullScreen</code> property.
+</description>
+<component-example>
+    <fd-dialog-mobile-example></fd-dialog-mobile-example>
+</component-example>
+<code-example [exampleFiles]="dialogMobile"></code-example>
+
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'complexDialog'" [componentName]="'dialog'">
+    Complex Dialog
+</fd-docs-section-title>
+<description>
+    To create more complex Dialogs it is possible to provide custom header, subheader and footer templates.<br/>
+    To provide custom template create element with on of the following directives:
+    <ul>
+        <li>Header - <code>fdTemplate="header"</code></li>
+        <li>Subheader - <code>fdTemplate="subheader"</code></li>
+        <li>Footer - <code>fdTemplate="footer"</code></li>
+    </ul>
+    Custom templates are embedded views for Bar Component so it is possible to use Bar Component features inside them.
+</description>
+<component-example>
+    <fd-dialog-complex-example></fd-dialog-complex-example>
+</component-example>
+<code-example [exampleFiles]="complexDialog"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'stackedDialogs'" [componentName]="'dialog'">
+    Stacked Dialogs
+</fd-docs-section-title>
+<description>
+    The DialogService supports limitless stacked dialogs. Each can be closed individually, there is no specific
+    closing order.
+    <br/><br/>
+    In the example below, it is accomplished via components as content. However, it is entirely possible with
+    templates through the same methodology.
```

Comment:
In the example below, this is accomplished by using components as content however this can also be achieved using  templates.